app/presenters/attachment_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class AttachmentPresenter
+
+  def initialize(attachment)
+    @attachment = attachment
+  end
+
+  def to_json
+    {
+      url: @attachment.url,
+      title: @attachment.title,
+      content_type: @attachment.content_type,
+      updated_at: updated_at,
+      created_at: created_at,
+      content_id: @attachment.content_id
+    }
+  end
+
+  private
```

Comment:
Oh and I think for consistency in this project I think private is indented 2 spaces behind everything else.